article.php

```diff
@@ -20,7 +20,11 @@
                 $con = database_connect($database);
                 $query_blog_article = 'SELECT `id`, `name`, `body`, `date_created` FROM `articles` WHERE `slug`="' . $_GET['blog'] . '";';
                 $article_data = query_into_array($con, $query_blog_article);
-                echo '<div class="blog-rows"><h2 class="blog-text">' . $article_data[0]['name'] . '</h2><h5 class="blog-text">' . $article_data[0]['date_created'] . '</h5><p id="blog-desc" class="blog-text">' . $article_data[0]['body'] . '</p></div>';
+                echo '<div class="blog-rows">' .
+                            '<h2 class="blog-text">' . $article_data[0]['name'] . '</h2>' .
+                            '<h5 class="blog-text">' . $article_data[0]['date_created'] . '</h5>' .
+                            '<p id="blog-desc" class="blog-text">' . $article_data[0]['body'] . '</p>' .
```

Comment:
It seemed not to like the fact that I had them separated over a line, but will re-check, thanks